backend/src/main/java/gov/cdc/usds/simplereport/api/model/User.java

```diff
@@ -1,32 +1,37 @@
 package gov.cdc.usds.simplereport.api.model;
 
+import java.util.Optional;
+import java.util.List;
+
 import gov.cdc.usds.simplereport.db.model.ApiUser;
 import gov.cdc.usds.simplereport.db.model.Organization;
 import gov.cdc.usds.simplereport.db.model.auxiliary.PersonName;
 
 public class User {
 
 	private String id;
-	private Organization org;
+	private Optional<Organization> org;
 	private PersonName nameInfo;
 	private String email;
 	private Boolean isAdmin;
+	private List<UserPermission> permissions;
```

Comment:
unrelated: are user types (admin, standard, or restricted) coming later?

Reply:
Two-part answer:
1. yes we are still expecting to use isAdmin, since that refers to _site_ administrators (e.g. us)
2. the user types you're looking for are in [OrganizationRole](https://github.com/CDCgov/prime-simplereport/blob/main/backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java).